### PR TITLE
Tidied up Streaming.py

### PR DESCRIPTION
**Pull request by Kevin-Gillanders**
Gave variables more descriptive names. made file PEP8 compliant

### CODE REVIEW

File: examples/streaming.py [hunk 1]
```diff
@@ -3,33 +3,42 @@
...
+auth.set_access_token(access_token, access_token_secret)
+api = API(auth)
 
 if __name__ == '__main__':
-    l = StdOutListener()
-    auth = OAuthHandler(consumer_key, consumer_secret)
-    auth.set_access_token(access_token, access_token_secret)
+    # Makes a MyStreamListener instance
+    myStreamListener = MyStreamListener()
+    myStream = Stream(auth = api.auth, listener = myStreamListener)
```
Comment (wjt) [inline examples/streaming.py]: If you're aiming to be PEP 8-compliant, there should not be spaces around the `=`s for keyword parameters. From the [Other Recommendations](https://www.python.org/dev/peps/pep-0008/#other-recommendations) section:

> Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.

Comment (fitnr) [inline examples/streaming.py]: Also see [the naming conventions section](https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments) for instances:

> lowercase with words separated by underscores as necessary to improve readability.

Comment (Harmon758) [inline examples/streaming.py:40]: As noted:
```suggestion
    stream = Stream(auth=api.auth, listener=stream_listener)
```

File: examples/streaming.py [hunk 2]
```diff
@@ -3,33 +3,42 @@
...
 
-    stream = Stream(auth, l)
-    stream.filter(track=['basketball'])
+    # In this example we will use filter to stream all tweets containing the word
+    # python. The track parameter is an array of search terms to stream.
```
Comment (Harmon758) [inline examples/streaming.py:43]:
```suggestion
    # "python". The track parameter is a list of search terms to stream.
```

File: examples/streaming.py [hunk 3]
```diff
@@ -3,33 +3,42 @@
...
+    myStream.filter(track = ['python'])
```
Comment (Harmon758) [inline examples/streaming.py:44]:
```suggestion
    stream.filter(track=['python'])
```

**Review by Harmon758 — CHANGES_REQUESTED** [review]
Using `API` isn't necessary for streaming and doesn't need to be added here.

**Comment by Harmon758** [comment]
This is stale now, with https://github.com/tweepy/tweepy/commit/e3e1d062377a5833b16e8ac8976a56ac57fac8a2.

**Pull request closed without merging**